actor/builtin/miner/miner.go

```diff
@@ -441,7 +434,7 @@ func (ma *Actor) CommitSector(ctx exec.VMContext, sectorID uint64, commD, commR,
 		return 1, errors.NewRevertError("invalid sized commRStar")
 	}
 
-	if !ma.Bootstrap {
+	if sectorID >= types.FirstNonGenGenSectorID {
```

Comment:
BLOCKING: sectorID is a parameter to CommitSector. This PR will prevent a node from sending a message with sectorID < FirstNonGenGenSectorID, but nothing prevents a miner from crafting such a message. I'm currently unclear whether this PR would allow miners to gain storage power without proving it.